jwst/white_light/tests/test_white_light.py

```diff
@@ -0,0 +1,110 @@
+from jwst import datamodels
+from jwst.white_light.white_light import white_light
+
+from astropy.time import Time, TimeDelta
+import math
+import numpy as np
+import pytest
+
+@pytest.fixture(scope='module')
+def make_datamodel():
+    """Make data for white light tests"""
+
+    model = datamodels.MultiSpecModel()
+    model.meta.exposure.group_time = 0.15904
+    model.meta.exposure.ngroups = 60
+    model.meta.exposure.start_time = 58627.0
+    model.meta.exposure.integration_start = 1
+    model.meta.exposure.integration_end = 2
+
+    # Make data arrays
+    flux = np.random.rand(200)*1e-9
+    wavelength = np.arange(11,13, step=0.01)
+    error = np.random.rand(200)
+    surf_bright = np.zeros(200)
+    sb_error = np.zeros(200)
+    dq = np.ones(200)
```

Comment:
Doesn't this set the dq to 'bad' ? I don't see them reset, so perhaps this doesn't matter if the dq values are never used.

Reply:
@dmggh -- Correct, the step doesn't actually uses the DQ flags but the `spec_table` attribute of the `SpectrumModel` is expecting all of the columns to be populated.